content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

```diff
@@ -0,0 +1,33 @@
+---
+title: kubeadm reset
+content_type: conceito
+weight: 60
+---
+<!-- overview -->
+Executa uma melhor reversão das alterações feitas pelo `kubeadm init` ou `kubeadm join`.
+
+<!-- body -->
+{{< include "generated/kubeadm_reset.md" >}}
+
+### Redefinindo o fluxo de trabalho {#reset-workflow}
+
+O `kubeadm reset` é o responsável por limpar um sistema de arquivos local dos nós a partir dos arquivos que foram criados usando os comandos `kubeadm init` ou `kubeadm join`. O `reset` dos nós do plano de controle também remove o etcd local do nó do cluster etcd.
+
+O `kubeadm reset phase` pode ser usado para executar separadamente as fases do fluxo de trabalho acima. Para pular uma lista de fases você pode usar `--skip-phases`, que funciona de maneira semelhante `kubeadm join` e `kubeadm init`.
```

Comment:
"phase runners" disappeared in the translation, maybe they could be added as "executores de fases dos comandos `kubeadm join`..."?